siamese_dataloader.py

```diff
@@ -25,8 +25,8 @@ def imshow(img,text=None,should_save=False):
 	if text:
 		plt.text(75, 8, text, style='italic',fontweight='bold',
 			bbox={'facecolor':'white', 'alpha':0.8, 'pad':10})
-	plt.imshow(np.transpose(npimg, (1, 2, 0)))
-	plt.show()    
+	plt.imshow(np.transpose(npimg, (1, 2, 0))) 
+	plt.savefig("figure.png")
```

Comment:
Saving figure instead of showing so that it can be used for future reference if required.